packages/api/src/schema.ts

```diff
@@ -2318,6 +2344,9 @@ const schema = gql`
     createGroup(input: CreateGroupInput!): CreateGroupResult!
     recommend(input: RecommendInput!): RecommendResult!
```

Comment:
I wonder if its better to just extend the `recommend` API? like `includeHighlights: bool`. At the time of recommending, does the backend have access to all the highlights?

Reply:
We have both methods to recommend highlights.
Just want to be more flexible. You can just use the original `recommend` api if you want